### ISSUE TITLE
Wrong scriptname request if django-shop run under a subfolder  

### ISSUE THREAD

**Issue by haricot** [awesto/django-shop#692]
This comes from django-cms, solved with this PR https://github.com/divio/django-cms/pull/6160

Otherwise it will be an error of the type:
`AttributeError("'NoneType' object has no attribute 'publisher_is_draft'",)`

**Comment by jrief** [comment]
How can I reproduce this?

**Comment by haricot** [comment]
Yes, if i use apache and mod_wsgi with WSGIScriptAlias /myshop without applying PR divio/django-cms#6160
I get the error above.

**Comment by haricot** [comment]
It happens when I go to an page url with an apphook Catalog List.
